transport/kubernetes-jobmonitor/src/test/kotlin/JobHandlerTest.kt

```diff
@@ -374,6 +376,63 @@ class JobHandlerTest : WordSpec({
 
             jobs shouldContainExactlyInAnyOrder listOf(matchJob1, matchJob2, matchJob3)
         }
+
+        "query only jobs for ORT Server workers" {
+            val coreApi = mockk<CoreV1Api>()
+            val jobApi = mockk<BatchV1Api>()
+
+            val jobList = V1JobList().apply { items = listOf(createJob("testJob")) }
+            every {
+                jobApi.listNamespacedJob(
+                    NAMESPACE,
+                    null,
+                    null,
+                    null,
+                    null,
+                    any(),
+                    null,
+                    null,
+                    null,
+                    null,
+                    false
+                )
+            } returns jobList
+
+            val handler = createJobHandler(jobApi, coreApi)
+            handler.findJobsCompletedBefore(OffsetDateTime.now())
+
+            val slotLabel = slot<String>()
+            verify {
+                jobApi.listNamespacedJob(
+                    NAMESPACE,
+                    null,
+                    null,
+                    null,
+                    null,
+                    capture(slotLabel),
+                    null,
+                    null,
+                    null,
+                    null,
+                    false
+                )
+            }
+
+            val labelSelectorRegex = Regex("""ort-worker in \((.+)\)""")
```

Comment:
If introducing a constant, it should be used here as well.

Reply:
In the test, I want to explicitly state the expectations. Therefore, I would not reuse a constant from the class under test.